flexmeasures/cli/data_add.py

```diff
@@ -941,6 +941,20 @@ def create_schedule(ctx):
     required=False,
     help="Optimize production against this sensor. Defaults to the consumption price sensor. The sensor typically records an electricity price (e.g. in EUR/kWh), but this field can also be used to optimize against some emission intensity factor (e.g. in kg CO₂ eq./kWh). Follow up with the sensor's ID.",
 )
+@click.option(
+    "--consumption-price-sensor-per-device",
+    "consumption_price_sensor_per_device",
+    type=dict,
+    required=False,
+    help="",
```

Comment:
Please specify help on what this command option is about. Probably text from our earlier discussions or the issue can be used to start.

Reply:
@nhoening, yes, i've added that to TODO.

Reply:
Addressed.